hbp_knowledge/proteostasis/brehme2016.bel

```diff
@@ -84,82 +84,108 @@ overall. However, in contrast with the broad spectrum of repressed
 #induced by aging
+SET Confidence = "High"
 bp(MESH:Aging) -| p(HGNCGENEFAMILY:"DNAJ (HSP40) heat shock proteins")
 bp(MESH:Aging) -| p(HGNC:HSPD1)
 bp(MESH:Aging) -| p(FPLX:HSPA)
 bp(MESH:Aging) -> p(HGNC:TPR)
 bp(MESH:Aging) -> p(FPLX:HSPB)
+UNSET Confidence
 
 SET Evidence = " Genetic approaches using
 these disease models demonstrate that the chaperome plays a crucial
 role in protecting cells from proteotoxicity."
+SET Confidence = "High"
 a(MESH:"Molecular Chaperones") -| path(HBP:proteotoxicity)
+UNSET Confidence
 
 SET Evidence = "Many studies based on model systems support a role for candidates
 from each of the major chaperome families; HSP100, HSP90,
 HSP70, HSP60, HSP40, sHSPs, and TPR-domain-containing
 proteins in proteostasis."
+SET Confidence = "Medium"
 bp(HBP:Proteostasis) -- p(HGNCGENEFAMILY:"DNAJ (HSP40) heat shock proteins")
 bp(HBP:Proteostasis) -- p(HGNC:HSPD1)
 bp(HBP:Proteostasis) -- p(FPLX:HSPA)
 bp(HBP:Proteostasis) -- p(HGNC:TPR)
 bp(HBP:Proteostasis) -- p(FPLX:HSPB)
-bp(HBP:Proteostasis) -- p(FPLX:HSPB)
 bp(HBP:Proteostasis) -- p(FPLX:HSP90)
 bp(HBP:Proteostasis) -- p(HGNCGENEFAMILY:"AAA ATPases")
+UNSET Confidence
 
 SET Evidence = "Early studies demonstrated that overexpression of a specific
 human HSP70 (HSPA1L) in a Drosophila disease model
 suppressed neurodegeneration associated with expression of
 polyQ-containing forms of both ataxin 3 or androgen receptor,
 and α -synuclein (Warrick et al., 1999; Chan et al., 2000, 2002;
 Auluck et al., 2002). "
+SET Confidence = "Medium"
+SET Species = "7227"
 p(HGNC:HSPA1L) -| path(HP:Neurodegeneration)
 path(HP:Neurodegeneration) -- p(HGNC:ATXN3, var("?"))
 path(HP:Neurodegeneration) -- p(HGNC:AR, var("?"))
 path(HP:Neurodegeneration) -- p(HGNC:SNCA, var("?"))
+UNSET Species
+UNSET Confidence
 
+#dominant - negative version?
 SET Evidence = " In a yeast model expressing the N-terminal
 fragment of a polyQ-containing huntingtin protein, overexpression
 of the yeast HSP70 (SSA1) reduced aggregate formation, whereas
 the dominant-negative version of the fly homolog to HSPA1L,
 Hsc4-K71S, enhanced neurodegeneration (Warrick et al., 1999;)"
+#SET Confidence = "Medium"
+SET Species = "4932"
 p(FPLX:HSPA) -| p(HBP:"huntingtin aggregates")
 p(HGNC:HSPA1L) -| path(HP:Neurodegeneration)
+UNSET Species
+#UNSET Confidence
 
 SET Evidence = " These studies revealed that an increase in levels of HSP70
 reduced aggregation of disease-associated proteins, thus playing a
 neuroprotective role."
+SET Confidence = "High"
 p(FPLX:HSPA) -| path(MESH:"Protein Aggregation, Pathological")
+UNSET Confidence
 
 SET Evidence = "For example,
 overexpression of the human TPR domain-containing co-
 chaperone CHIP suppresses neurodegeneration in fly models
 expressing polyQ-containing versions of ataxin 1 and the
 N-terminal huntingtin fragment (Al-Ramahi et al., 2006)."
 #STUB1 is CHIP
+SET Confidence = "Medium"
 p(HGNC:STUB1) -| path(HP:Neurodegeneration)
+p(HGNC:STUB1) -| p(HGNC:ATXN3, var("?"))
```

Comment:
This is not mentioned here!! (+ statement below)
You can only suggest this from the evidence, so i would not add the statements.